lib/options-manager.js

```diff
@@ -289,6 +290,9 @@ const buildESLintConfig = options => config => {
 		};
 	}
 
+	// Resolve plugins from XO root directory
+	config.resolvePluginsRelativeTo = MODULE_DIRECTORY;
```

Comment:
Doesn't this make it impossible for users to add ESLint plugins themselves?

Reply:
I tested in `eslint-plugin-unicorn`, it can load `eslint-plugin-eslint-plugin`.